client/src/components/ReviewList.jsx

```diff
@@ -1,16 +1,91 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import credentials from '../../../config.js'
+// import API from '../../../server/router.js'
+import Review from './Review.jsx'
 
 class ReviewList extends React.Component {
   constructor(props) {
     super(props)
     this.state = {
-      reviews: []
+      product: "",
+      reviews: [],
     }
+    this.getProductList = this.getProductList.bind(this);
+    this.getReviews = this.getReviews.bind(this);
+    this.updateState = this.updateState.bind(this);
   }
+
+  componentDidMount() {
+    this.getProductList()
+    this.getReviews(16057)
+  }
+
+  getProductList() {
+    axios({
+      method: 'GET',
+      url: 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-lax/products/',
+      headers: { 'Authorization': credentials.TOKEN },
```

Comment:
Last comma in headers might throw something

Reply:
Also were planning on doing the Api requests via server later right? Might need to change later

Reply:
yes--will definitely refactor later to do API calls to server.